scripts/emerge/ingest_to_tdr.py

```diff
@@ -72,11 +74,11 @@ def ingest_dataset(dataset_id, data):
     return json.loads(response.text)
 
 
-def create_ingest_dataset_request(control_file_path, target_table_name, load_tag=None):
+def create_ingest_dataset_request(ingest_records, target_table_name, load_tag=None):
```

Comment:
do you want to refactor the variables/inputs for `call_ingest_dataset` as well?

line 95: `call_ingest_dataset(control_file_path, ...`